.tekton/scanner-db-pull-request.yaml

```diff
@@ -239,12 +245,22 @@ spec:
           - name: basic-auth
             workspace: git-auth
 
+      - name: determine-image-tag
+        runAfter:
+        # This task must run on a freshly cloned repository to prevent seeing any changes from other tasks.
+        - clone-repository
+        taskRef:
+          name: determine-image-tag
+        workspaces:
+        - name: source
+          workspace: workspace
+
       - name: prefetch-dependencies
         params:
           - name: input
             value: $(params.prefetch-input)
         runAfter:
-          - clone-repository
+          - determine-image-tag
```

Comment:
Please update `fetch-sql-definitions` the same way here and in all pipelines where it or similar is present.

Reply:
done.